client/src/components/ExpenseAdder/ExpenseAdder.tsx

```diff
@@ -10,30 +10,56 @@ import {
   Label,
   Input,
   FormButton,
+  ErrorContainer,
 } from "./ExpenseAdder.styles";
 import { MdOutlineCancel } from "react-icons/md";
-import { ExpenseData } from "../../pages/Expenses/Expenses";
+import { useForm, SubmitHandler } from "react-hook-form";
+import * as yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { addExpenseItem } from "../../API/ExpenseMethods";
+
+const schema = yup.object().shape({
```

Comment:
I've moved the Schema to the constants file for now even though it's only been used once since all income/expense has been consolidated - let me know if I shoulld just move it back